style.css

```diff
@@ -0,0 +1,297 @@
+  }
+
+  .Q{
+    transform: translate(5px, 39px) rotate(-10deg);
+   
+  }
+  .U{
+    transform: translate(5px, 25px) rotate(-9deg);
+   
+  }
+  .I{
+    transform: translate(5px, 19.5px) rotate(-7.5deg);
+   
+  }
+  .Z{
+    transform: translate(5px, 16px) rotate(-5deg);
+   
+  }
+  .M{
+    transform: translate(5px, 14px) rotate(0deg);
+   
+  }
+  .I1{ 
+      transform: translate(5px, 14px) rotate(0deg);
+  }
+  .N{
+    transform: translate(5px, 16px) rotate(4deg);
+   
+  }
+  .I2{
+    transform: translate(5px, 19.5px) rotate(7.5deg);
+   
+  }
+  .A{
+    transform: translate(5px, 25px) rotate(9deg);
+   
+  }
+  .C{
+    transform: translate(5px, 39px) rotate(10deg);
+   
+  }
+
+.category-head{
+    font-size: 2rem;
+    width: 80%;
+    padding: 1rem;
+    color: gold;
+    text-shadow: 2px 2px 2px yellow;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+}
+.category-head a{
+    margin-left: auto;
+    color: white;
+    font-size: 1.5rem;
+}
+.category-cont{
+    
+    width: 80%;
+    height: 100%;
+}
+.category-cont .category-subhead{
+    
+    padding: 1rem;
+    font-size: 1.5rem;
+    color: white;
+    border-bottom:solid;
+    margin-bottom: 1rem;
+}
+
+.category-cont .category-menu{
+    
+    display: flex;
+    justify-content: space-evenly;
+    padding: 0rem;
+}
+.category-cont .category-menu .menu-item{
+    color: white;
+    padding: 1rem;
+}
+.category-cont .category-menu .menu-item:nth-child(2){
+    border-bottom: 2px solid yellow;
+    color: yellow;
+}
+
+.category-item-cards{
+    display: grid;
+    grid-template-columns:1fr 1fr;
+    grid-template-rows: 1fr 1fr;
+    margin: 2rem 5rem 5rem 5rem;
+    gap: 1rem;
+}
+.card-item{
+    min-width: 10rem;
+    height: 15rem;
+    border-radius: 1rem;
+    display: flex;
+    position: relative;
+    flex-direction: column;
+    transition: ease-in 0.5;
+}
+
+.card-item:hover{
+    box-shadow: 1px 1px 1px 2px yellow;
+}
+
+.card-item .img-cont{
+    height: 100%;
+    
+}
+.card-item .img-cont img{
+    max-inline-size: 100%;
+    border-radius: 1rem;
+    block-size: auto;
+    aspect-ratio: 4/3;
+    object-fit: contain/cover;
+    object-position: top center;
+    height: 100%;
+    width: 100%;
+    object-fit: cover;
+    transition: 0.5s;
+}
+.card-item:hover .img-cont img{
+    max-inline-size: 100%;
+    block-size: auto;
+    aspect-ratio: 4/3;
+    object-fit: contain/cover;
+    object-position: top center;
+    height: 100%;
+    width: 100%;
+    filter:blur(1px) brightness(0.5);
+    object-fit: cover;
+}
+
+.card-item .btn-cont{
+    visibility: hidden;
+}
+
+.card-item:hover .btn-cont{
+    position: absolute;
+    visibility: visible;
+    top: 50%;
+    left: 45%;
+}
+.card-item .btn-cont a{
+    all: unset;
+    padding: 1rem;
+    color: white;
+    background-color: #221C69;
+    border-radius: 1rem;
```

Comment:
you have used color names like yellow black white at all places, replace them with hex codes to maintain consistency in color values